cost-analyzer/templates/aggregator-statefulset.yaml

```diff
@@ -79,6 +81,9 @@ spec:
         {{- if .Values.kubecostAggregator.containerSecurityContext }}
           securityContext:
             {{- toYaml .Values.kubecostAggregator.containerSecurityContext | nindent 12 }}
+        {{- else if .Values.global.containerSecurityContext }}
+          securityContext:
+            {{- toYaml .Values.global.containerSecurityContext | nindent 12 -}}
```

Comment:
are you good with the trailing hyphen here? Will that pull the next key in to the back of this?

Reply:
Thanks for calling that out. It won't have a rendering effect but it shouldn't be there. This was a copy-and-paste artifact from another template (that should have also been removed). Pushed that now.